src/Pill.ino

```diff
@@ -62,6 +94,13 @@ void loop() {
   int currentSecond = Time.second();
   if (currentSecond != lastActiveSecond) {
     lastActiveSecond = currentSecond;
+    if (blinkLed) {
+      if (ledIsOn) {
+        ledOff();
+      } else {
+        ledOn();
```

Comment:
eh? it'll only turn on when blinked?

Reply:
the night time event is what turns it on, this is only if blinking is active